Edits to About DevOps Dojo (#53)
dxc-technology/about-devops-dojo: blog/_posts/2020-04-14-introducing-online-devops-dojo.md

excerpt: "Introducing Online DevOps Dojo: an immersive, Open Source learning experience with a fictious team and a set of hands-on labs running in the browser."
tags: [devops, learning]
---

## DevOps Dojos

DevOps Dojos became popular since Target, an important U.S.-based retail company, started to introduce such a practice back in 2014 in support of its DevOps transformation.
Since then, multiple conference talks, white papers and — more recently — books have been published about DevOps Dojos.

Back in 2015, we created our own DevOps Dojos, which we have described in multiple talks - [1](https://www.youtube.com/watch?v=nhokY2UpBqI){:target="_blank" rel="noopener"}, [2](https://www.youtube.com/watch?v=75aZRf0_wSo){:target="_blank" rel="noopener"}, [3](https://www.youtube.com/watch?v=5ejcK1sTv6k){:target="_blank" rel="noopener"} - in conferences and meetups, sharing our own experience.

Since then, our own employees and customers alike have been taking advantage of the DevOps Dojo.

Yet, we quickly came to a point where our physical DevOps Dojos were not enough to cover all the needs. Everyone was loving the

> experience, and to this day the on-site expereince still leads to great results. But we had a scale issue: we just could not accommodate everyone we needed to address.

Reviewer: ```suggestion
experience, and to this day the on-site experience still leads to great results. But we had a scale issue: we just could not accommodate everyone we needed to address.

```

The modules tell the story of a fictious company, "Pet Clinic," and its employees as they go through their DevOps journey.

![Once Upon a Time](../../images/onceuponatime.jpg)

Let us introduce you to some of the team members:

<img src="../../images/brenda.png" width="400"/> | **B**renda from the **B**usiness, who is keen to take the Pet Clinic into new growth areas, sees DevOps being key to that. She is extremely proud of her two golden retrievers Bonnie and Clyde.
![Chun](../../images/chun.png) | **C**hun is a **c**oach, change facilitator and technologist. Chun is an avid outdoors person and adrenaline junky, lover of fun, hater of being on-call. She encourages teams to automate the build, test, package and deploy process as much as possible of to ensure that the systems are created with a LessOps model in mind.
![Dan](../../images/dan.png) | **D**an has worked as a senior developer for a number of years, most recently for First Order Incorporated, a role he left due to the excessive travel requirements. Dan's workmates still fondly recall the fun and games that ensued when Shelob, his pet tarantula, got loose on *"Bring Your Pets to Work"* day.

> ![Adam](../../images/adam.png) | **A**dam is a System **A**dministrator transitioning to the role of Site Reliability Engineer (SRE). Adam is working closely with Selma to ensure the security of the company's software and applications. Adam suspects that Hal may have an inside man. Adam has a particular set of skills so he is working on a plan to look for the hacker, to find the hacker, to ... the hacker.

Reviewer: ```suggestion
![Adam](../../images/adam.png) | **A**dam is a System **A**dministrator transitioning to the role of Site Reliability Engineer (SRE). Adam is working closely with the Security Officer to ensure the security of the company's software and applications. Adam has a particular set of skills so he is working on a plan to look for the hacker, to find the hacker, to ... the hacker.

```

Reviewer: Thanks a lot Lisa. Selma and Hal are described in the home page: https://dxc-technology.github.io/about-devops-dojo/
Instead of making another long list of personas in the blog post, I have removed the references to Selma and Hal names.
